zipline/pipeline/loaders/blaze/core.py

```diff
@@ -952,6 +954,17 @@ def collect_expr(e):
                 columns=added_query_fields + list(map(getname, columns)),
             )
         )
+        if not materialized_deltas.empty and have_sids:
+            missing_sids = materialized_deltas[
+                ~materialized_deltas.sid.isin(assets)
+            ]
+            log.debug(
+                "Didn't find the following sids in asset index: {}".format(
```

Comment:
Right now, the only cases where this happens is with invalid sids in the dbz database. E.g., there's a symbol that we symbol mapped incorrectly because we used a stale assets db. There's another symbol that a vendor is sending us updates for that expired years ago. Basically, all the times that this has been failing currently give us info that we probably want to know about. Maybe there's a better way/place to do this, though.